app/views/admin/budget_headings/update.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if @errors %>
```

Comment:
Hopefully using the above partial for errors and always painting it will avoid having to manually display it, but not sure... let's see...